repository/src/main/java/org/alfresco/repo/domain/permissions/AclDAO.java

```diff
@@ -182,4 +182,11 @@
      * @return Long
      */
     public Long getMaxChangeSetIdByCommitTime(long maxCommitTime);
+
+    /**
+    * @return                 the commit time of the current ACL change set entry or <tt>null</tt> if
+    *                         there have not been any modifications.
+    */ 
+    Long getCurrentChangeSetCommitTime();

```

Comment:
The method should be "public", right?

Reply:
in interfaces, the default modifier is public, so as IntelliJ uses to say: "Modified ‘public’ is redundant for interface methods" :)

Reply:
Please make it consistent with the rest of the interface.